plugins/modules/swim_intent.py

```diff
@@ -50,26 +50,26 @@
         type: dict
         suboptions:
           type:
-            description: The source of import, supports url import or local import.
+            description: The source of import, supports remote import or local import.
             type: str
           local_image_details:
             description: Details of the local path of the image to be imported.
             type: dict
             suboptions:
               file_path:
-                description: File absolute path.
+                description: Give the file absolute path required while importing image from local.
```

Comment:
description: The absolute path of the file required for local import.
example: "/path/to/your/file"

OR
description: Provide the absolute file path needed to import an image from your local system.

Reply:
added.